ksqldb-rest-app/src/test/java/io/confluent/ksql/api/AuthTest.java

```diff
@@ -539,7 +612,7 @@ private void shouldNotAllowAccessIfPermissionCheckThrowsException(
     void run() throws Exception;
   }
 
-  private static class StringPrincipal implements Principal {
+  public static class StringPrincipal implements Principal {
```

Comment:
Why can't this still be private?

Reply:
Left from another approach I was taking. Fixed.